benchmark/src/main/scala/mesosphere/marathon/core/storage/repository/TemplateRepositoryBenchmark.scala

```diff
@@ -0,0 +1,202 @@
+package mesosphere.marathon
+package core.storage.repository
+
+import java.util.concurrent.TimeUnit
+
+import akka.actor.ActorSystem
+import akka.stream.ActorMaterializer
+import akka.stream.scaladsl.{Sink, Source}
+import com.typesafe.scalalogging.StrictLogging
+import mesosphere.marathon.core.storage.repository.TemplateRepositoryLike.{Spec, Template}
+import mesosphere.marathon.core.storage.repository.impl.{CachedTemplateRepository, TemplateRepository}
+import mesosphere.marathon.core.storage.zookeeper.{AsyncCuratorBuilderFactory, AsyncCuratorBuilderSettings, ZooKeeperPersistenceStore}
+import mesosphere.marathon.metrics.Metrics
+import mesosphere.marathon.metrics.dummy.DummyMetrics
+import mesosphere.marathon.state.{AppDefinition, PathId}
+import org.apache.curator.framework.{CuratorFramework, CuratorFrameworkFactory}
+import org.apache.curator.retry.BoundedExponentialBackoffRetry
+import org.apache.curator.x.async.api.CreateOption
+import org.openjdk.jmh.annotations._
```

Comment:
nit: no to `_` imports, if possible.

Reply:
I'm generally inclined to agree. This is benchmark code, though, and this is for annotations. I might let this one slide.

Reply:
There are 12 imported annotations there. Is it really better to import them one by one? But if you insist...

Reply:
I side with Ivan. It's not easy to track the annotations and it's less than a dozen 😉 .